docs/rfc/rfc-1.adoc

```diff
@@ -370,8 +370,8 @@ Governable Parameters:
 
 - `redemption_request_timeout`: The amount of time the wallet has to provide
   redemption proof.
-- `redemption_request_timeout_slashing_amount`: The amount of stake slashed from
-  each member of a wallet for a timed-out redemption request.
+- `redemption_request_timeout_slashing_amount`: The amount of stake slashed
+  from each member of a wallet for a timed-out redemption request.
 - `redemption_request_timeout_redemeer_bonus_multiplier`: The percentage of the
   notifier reward from the staking contract the redeemer receives in case of a
   redemption timeout.
```

Comment:
Later in file ([#L493](https://github.com/keep-network/tbtc-v2/blame/9a22947d00d5d36f0ba4ba4b864c2cfebe06a3f5/docs/rfc/rfc-1.adoc#L493)) we call this parameter `redemption_request_timeout_redemeer_bonus`. Could we fix that?
Also, shouldn't the explanation of this param be:
"The percentage of the slashed stake the notifier receives in case of a redemption timeout."
?
If so, there would be some change needed also in the "Redemption timeout" section.